egress_proxy/variables.tf

```
@@ -0,0 +1,64 @@
+variable "cf_org_name" {
+  type        = string
+  description = "cloud.gov organization name"
+}
+
+variable "cf_space_name" {
+  type        = string
+  description = "cloud.gov space name for egress (eg staging-egress or prod-egress)"
+}
+
+variable "client_space" {
+  type        = string
+  description = "cloud.gov space name for client apps (eg staging or prod)"
+}
+
+variable "name" {
+  type        = string
+  description = "name of the egress proxy application"
+}
+
+variable "egress_memory" {
+  type        = number
+  description = "Memory in MB to allocate to egress proxy app"
+  default     = 64
+}
+
+variable "gitref" {
+  type        = string
+  description = "gitref for the specific version of cg-egress-proxy that you want to use. Branch name should start with `refs/heads` while a git sha are given without a prefix"
```

some mismatched pluralization I think?
```suggestion
  description = "gitref for the specific version of cg-egress-proxy that you want to use. Branch name should start with `refs/heads` while a git sha should be given without a prefix"
```